.gitignore

```diff
@@ -1 +1,2 @@
 bin
+go.sum
```

Comment:
Isn't this a bad idea?
https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file

Reply:
I'm wondering if I better trust the values in the proxy instead though. If anything, we should at least add a check that the committed go.sum file is correct.

Reply:
Maybe we should ensure go mod tidy results in a clean go.sum (add it to our ci/cd)
I'm not comfortable ignoring the go.sum file.

Reply:
That sounds good to me. I'll close this out